src/s390_ecc.c

```diff
@@ -396,10 +396,11 @@ static short get_default_domain(void)
 		return domain;
 
```

Comment:
Lines 391 and 392: Isn't this dead code? Yit define variable domain in line 387 and initialize it to -1. 
The if in line 391 will never be true......

Reply:
Yes, definitely. It's old code, but good to fix, maybe in a separate commit.